### ISSUE TITLE
Supporting you

### ISSUE THREAD

**Issue by miathedev** [archnemesis/uconsole-os#1]
Is there anyway i could support you with this project?
I own a uConsole for a week now.

I already tryed to compile your project, without luck at the QT5 build process.
Anyways, if i can support you somehow, with e.g. a Build Server, let me know.

With love,
Mia

**Comment by archnemesis** [comment]
I wouldn't be surprised if you just hit out-of-memory issues while compiling Qt. Sometimes if you just do another "./dmake all" again it'll pick up where it left off and continue normally. Try giving your build host as much RAM as possible, maybe even add some swap space too. I usually hit the same problem, where the compilation looks like it just fails for no reason. Compiling qtwebengine is especially memory-intensive.

**Comment by miathedev** [comment]
Thank you for the QT5 hint! Got it built and flashed.

The Pi successfully booted, but unfortunately no display output, only HDMI.
Is it correct, that only the back-light works right now?
So that i know if i broke something somehow.

**Comment by archnemesis** [comment]
Yeah for some reason after a cold startup the LCD screen isn't working, but if you do a soft reboot it does. I think there is some kind of timing and/or power sequencing issue going on, but I have not been able to figure it out. I have a feeling that building the LCD driver as a module may work better but I haven't tested it. My theory is that the LCD commands are getting sent before the LCD screen is ready, so maybe delaying the loading of the driver until the modules are loaded may help. I haven't checked recently to see if anyone has solved the issue, but I did read some reports about the LCD screen being flaky.

**Comment by miathedev** [comment]
I can confirm the soft reboot works.
What a strange thing.

Im getting my feet wet and try to take a look at it too on weekend.
I already splited the chunky uconsole patch into multiple using splitpatch for better readability. That uconsole patch is a big monstrosity!

Please let me know when you got any news. Im not into Linux Development that super deep as you do. But ill try my best to try things out and hopefully help.
